conf/xml/applications/HumanDynamicsEstimation-HumanKinematics.xml

```diff
@@ -0,0 +1,49 @@
+<application>
+  <name>HumanDynamicsEstimation-HumanKinematics</name>
+  <description>An application for running Human Dynamics Estimation (HDE) for only human</description>
+
+  <module>
+    <name>yarprobotinterface</name>
+    <parameters>--config TransformServer.xml</parameters>
+    <node>localhost</node>
+  </module>
+  
+  <!--yarprobotinterface with HDE config file for only human-->
+  <module>
+    <name>yarprobotinterface</name>
+    <parameters>--config HumanStateProvider.xml</parameters>
+    <dependencies>
+          <port timeout="5.0">/transformServer/rpc</port>
+          <port timeout="5.0">/transformServer/transforms:o</port>
+          <port timeout="5.0">/XSensSuit/WearableData/data:o</port>
+          <port timeout="5.0">/FTShoeLeft/WearableData/data:o</port>
+          <port timeout="5.0">/FTShoeRight/WearableData/data:o</port>
+    </dependencies>
+    <environment>YARP_FORWARD_LOG_ENABLE=1</environment>
+    <description>Run HDE for only human</description>
+    <node>localhost</node>
+  </module>
+
+  <!--yarprobotstatepublisher for human-->
+  <module>
+    <name>yarprobotstatepublisher</name>
+    <parameters>--period 0.0001 --name-prefix Human --tf-prefix /Human/ --model humanSubject01_66dof.urdf --reduced-model true --base-frame Pelvis --jointstates-topic "/Human/joint_states"</parameters>
+    <dependencies>
+      <port timeout="5.0">/Human/joint_states+@/HumanStatePublisher</port>
+      <port timeout="5.0">/tf+@/HumanStatePublisher</port>
+    </dependencies>
+    <environment>YARP_FORWARD_LOG_ENABLE=1</environment>
+    <description>Run yarprobotstatepublisher with correct human model</description>
+    <node>localhost</node>
+  </module>
+  
+  <!--Rviz-->
+  <module>
```

Comment:
@lrapetti May be adding dependencies of ports opened by HDE devices that stream data through topics for Rviz is helpful.

Reply:
I checked and RViz but its own can be launched even before the other applications are running (and then once the other applications run you start visualizing the model), so there is no need of waiting for the other applications to stream the data before running RViz.

Indeed for the time being I would proceed merging as it is, and eventually we can add later the dependencies if we see the need for them.

Reply:
> I checked and RViz but its own can be launched even before the other applications are running (and then once the other applications run you start visualizing the model)

this is true, but in the past I remember @kouroshD was launching the application and HDE related devices were crashing with Rviz model collapsed. This can be a source of confusion for new users trying to launch the yarp application. 

> Indeed for the time being I would proceed merging as it is, and eventually we can add later the dependencies if we see the need for them.

Agree with this 👍🏾

Reply:
> I remember @kouroshD was launching the application and HDE related devices were crashing with Rviz model collapsed. This can be a source of confusion for new users trying to launch the yarp application.

sorry, I was not aware of this, then we should investigate it!